auth-foundation/src/main/java/com/okta/authfoundation/client/internal/NetworkUtils.kt

```diff
@@ -131,19 +131,20 @@ private suspend fun OidcConfiguration.executeRequest(
     var response = okHttpClient.newCall(request).await()
 
     if (response.code == rateLimitErrorCode) {
-        val event = RateLimitExceededEvent(request, response)
+        var event = RateLimitExceededEvent(request, response)
```

Comment:
I thought we were doing this for very request before! Whoops!

Reply:
No worries! I missed this also